### DISCUSSION TITLE
OpenWRT stuck

### DISCUSSION THREAD

**Discussion by limendoza** [asylumexp/Proxmox#91] [Q&A]
I tried running the OpenWRT script and it gets stuck at "Network interfaces have been successfully configured". I'm running it on a Raspberry Pi 5.

**Comment by opekb** [comment]
![image](https://github.com/user-attachments/assets/36551a0d-8a2a-4bba-8a5a-9e1ceca4b61d)

**Comment by asylumexp** [comment]
I’m unable to do any work on this until after December 10th, after then I’ll try get this working.

**Comment by ibrah3m** [comment]
I successfully running OpenWrt after 12H of try and failure.

I don't remember all the setps I did but Does making template helps ?

**Comment by opekb** [comment]
@asylumexp Good afternoon! Couldn't verify the work?

**Reply by asylumexp** [reply]
I’ve been busy with life stuff recently, when I’ve got time I’ll get it fixed

**Comment by asylumexp** [comment]
can someone test with latest commit?

**Reply by opekb** [reply]
It doesn't work.

 ✗ This script will not work with PiMox!

**Reply by asylumexp** [reply]
oops merged upstreams massive changes and must have overwrote my script I’ll go back later and try fix

**Reply by opekb** [reply]
Good afternoon! Haven't checked it yet?

**Reply by asylumexp** [reply]
> Good afternoon! Haven't checked it yet?

https://github.com/asylumexp/Proxmox/issues/101#issuecomment-2594676940

Can you tell me whether this works or not? I can’t test VMs and haven’t been told whether this works or not by anyone yet.

**Reply by opekb** [reply]
The script works without errors. But the console is not available through the pimox interface. Also, the created virtual machine is not rebooted using standard pimox tools.

**Reply by asylumexp** [reply]
> The script works without errors. But the console is not available through the pimox interface. Also, the created virtual machine is not rebooted using standard pimox tools.

does it show anything via webui?

**Comment by opekb** [comment]
![Screenshot 2025-02-05 115420](https://github.com/user-attachments/assets/1acf19b3-b67d-414f-b13b-fbd321786cc6)
![Screenshot 2025-02-05 115451](https://github.com/user-attachments/assets/de24ce73-5d2d-4ee9-9e9c-9ff8c6197ca8)
The console does not respond to keystrokes.
